### ISSUE TITLE
About the performance of model trained on market1501

### ISSUE THREAD

**Issue by wangyingquan** [huanghoujing/beyond-part-models#35]
Thanks for your work, I'm sorry to bother you.
I try to train the PCB on marker1501 , but the performance I get is very different from yours ( cmc1=76.69% cmc5=85.72%).At the same time , I used the 'model_weight.pth' you provided to test the models can reach the performance  about cmc1=93.02% .Do you know the reason?

**Comment by huanghoujing** [comment]
Hi, thanks for your attention. Do you use Python 2.7 and Pytorch 0.3?

**Comment by wangyingquan** [comment]
Yes.But I forgot that I modified a line of code . Now the performance of model is almost same as you show . Thank you for your reply !
